### PR TITLE
Assignment 1: Nicholas Burlakov/Dillon Pikulik

### PR DESCRIPTION
**Pull request by NicholasBurlakov**
Azure link: https://ambitious-grass-04916f41e.3.azurestaticapps.net
Extra credit ideas are in README and here too because why not:
1. A hint system where you can get help like, "use this word in a sentence".
2. A strategy guide to the most effective words that use the most vowels/letters. (Or a link to r/Wordle)
3. A timed mode where you get one minute to answer the most words possible.

### CODE REVIEW

**Comment by DominicAlvarez** [comment]
Create a Vue app ❌✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest✅
Create a simple page called game.vue ❌

Page should have a card with a heading that with a clever name for the game  some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest tests run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ❌
Extra Credit
Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ✅
One point per idea up to three points

Missing a assignment 1 branch, the pull request was made with the main branch. App is also the same as the wordle projectwe have been working on in class. I think it was supposed to be its own separate vue app. Other than needing another branch it looks fine, and it is combined with the work weve been doing in class rather than its own separate thing.

File: wordle-web/src/views/game.vue [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<template>
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:1]: This file name should be PascaleCase

File: wordle-web/src/views/game.vue [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  <div>
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:2]: You don't need this div

File: wordle-web/src/views/game.vue [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      <v-card-title>Scrabble But Better</v-card-title>
+      <v-card-subtitle> This is what we want to build: </v-card-subtitle>
+
+      <v-card-text>
+        We are here to make a better, more clever version of Scrabble.
+      </v-card-text>
+
+      <v-card-actions>
+        <v-spacer />
+        <RouterLink to="/">Home<v-btn></v-btn> </RouterLink>
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:13]:
```suggestion
        <v-btn to="/"> Home </v-btn>
```

**Review by mmwoodfo — COMMENTED** [review]
This PR should be against our assignment 1 branch, not against main

**Pull request closed without merging**